compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/res/DesktopSvgResources.desktop.kt

```diff
@@ -49,6 +49,7 @@ import org.jetbrains.skia.svg.SVGPreserveAspectRatioAlign
  * will be drawn with the specified size, density will have no effect.
  * @return the decoded SVG image associated with the resource
  */
+@Deprecated("Migrate to the compose resources library")
```

Comment:
Also, we need to check if the new resources can be used in our current/potential client environments (Toolbox, InteliJ, Android Studio repos). Not sure that they apply `compose-multiplatform` Gradle plugin. Could you investigate this?

Reply:
Since compose 1.7.0 there are functions to convert xml, svg, png etc. byte arrays:
```kotlin
fun ByteArray.decodeToImageBitmap(): ImageBitmap
fun ByteArray.decodeToImageVector(density: Density): ImageVector
fun ByteArray.decodeToSvgPainter(density: Density): Painter
```

if an app doesn't use gradle plugin it is still able to read a plain java resource as the byte array and convert it to the compose object

Reply:
Let's write a replacement function in the deprecation comment, to be completely sure, and provide a fast solution for non-Gradle users

Reply:
@terrakok, I am waiting for this to approve the PR.

Reply:
We also need a replacement for `painterResource`, if users want to load specifically from classpath (Jewel, IntelliJ platform, Swing applications).

I suggest do it this way:

1. Add "If you need to load resources specificly from Java classpath, you should still use the new resource library and use a snippet from https://github.com/JetBrains/compose-multiplatform-core/pull/1457"
2. Add a snippet in the PR description. Something like this:
```
@Composable
fun bitmapPainterResource(path: String): Painter =
    remember { BitmapPainter(readResourceBytes(path).decodeToImageBitmap()) }

@Composable
fun imageVectorPainterResource(path: String): Painter {
    val imageVector = remember { readResourceBytes(path).decodeToImageVector() }
    return rememberVectorPainter(imageVector)
}

@Composable
fun svgPainterResource(path: String): Painter =
    remember { readResourceBytes(path).decodeToSvgPainter() }

fun readResourceBytes(resourcePath: String) =
    ClassLoader.getSystemClassLoader().getResourceAsStream(resourcePath).readAllBytes()
```
(please, check correctness)